src/mol-plugin-ui/skin/base/components/viewport.scss

```diff
@@ -173,15 +173,13 @@
     background: rgba(red($default-background), green($default-background), blue($default-background), 0.5);
 
     position: absolute;
+    height: 50vh; // 50% of the viewport height
```

Comment:
I think this will break it in the normal viewer. Should this still be a `max-height`? 

Perpahs consider creating a separate class just for Mesoscale explorer needs? (can still add fixes to the default one)

Reply:
could you point me on how to create a custom msp class like 'msp-state-snapshot-viewport-controls'. Where should it go ? in style.scss in the mesoscale-explorer folder ? or should it go in. a new scss file ?